music21/harmony.py

```diff
@@ -2492,6 +2493,32 @@ def transpose(self: NCT, _value, *, inPlace=False) -> NCT | None:
 
 # ------------------------------------------------------------------------------
 
+class FiguredBassIndication(Harmony):
+    isFigure = True
+    tstamp = 1
+    def __init__(self, figs=None, **keywords):
```

Comment:
It would be good to have type annotations for the `__init__` here. `figs: ?? | None = None`

Reply:
type annotation is added.